package.py

```diff
@@ -47,6 +47,7 @@ def process_ontology(in_dir: str, out_dir: str, version: str):
     opencs.serialize_multi(g, out_dir + '/opencs_header')
     opencs.parse_all(g, in_dir + '/ontology/core/**/*.ttl')
     g.parse(in_dir + '/ontology/authors.ttl')
+    opencs.validate_shacl(g)
```

Comment:
Please put this into a completely separate script, like "validate.py".

Please also add a new stage to OpenCS's CI pipeline for validation. It should probably come BEFORE packaging, not after it.

Reply:
To elaborate more on the "why" – someone might want to switch off validation for some pipelines, for example. Or we might switch to a different validator at some point. This would all be much, much simpler if we didn't combine the scripts like that.

The scripts should more-or-less follow the Unix philosophy of doing **one** thing, but doing it well :)